### PR TITLE
Dev merge, release candidate

### PR DESCRIPTION
**Pull request by jsbnr**
Adds a number of features including place holder images and tidies and fixes a number of other issues. Added and refactored documentation to ease setup.

### CODE REVIEW

File: visualizations/podium/components/PodiumPlace.tsx [hunk 1]
```diff
@@ -1,77 +1,145 @@
...
+  const placingOffsetWidth = 0.8;
 
   let imageThumbnail, imageThumbnailCircle;
-  if(imageUrl && imageUrl!="")  {
-    imageThumbnail=<div className="imageThumb"><img src={imageUrl} alt="name"/></div>
-    imageThumbnailCircle=<div className="imageThumb"><img className="circle" style={{borderColor: decorationColor}} src={imageUrl} alt="name"/></div>
+  if (imageUrl && imageUrl != "") {
+    imageThumbnail = (
+      <div className="imageThumb">
+        <ImageWithFallback src={imageUrl} alt={name} />
```
Comment (matewilk) [inline visualizations/podium/components/PodiumPlace.tsx]: `name` prop is not being passed down or provided by a hook

File: visualizations/podium/components/PodiumPlace.tsx [hunk 2]
```diff
@@ -1,77 +1,145 @@
...
+      </div>
+    );
+    imageThumbnailCircle = (
+      <div className="imageThumb">
+        <ImageWithFallback
+          src={imageUrl}
+          alt={name}
```
Comment (matewilk) [inline visualizations/podium/components/PodiumPlace.tsx]: same here

**Review by matewilk — CHANGES_REQUESTED** [review]

File: visualizations/podium/components/PodiumPlace.tsx (hunk 1 above)
Comment (jsbnr) [reply visualizations/podium/components/PodiumPlace.tsx]: done

File: visualizations/podium/components/PodiumPlace.tsx (hunk 2 above)
Comment (jsbnr) [reply visualizations/podium/components/PodiumPlace.tsx]: done

**Review by matewilk — APPROVED** [review]

**Pull request merged**